shared/js/settings.js

```diff
@@ -1,143 +1,143 @@
-var load = require('load');
+var load = require('./load');
 /**
  * Public api
  * Usage:
  * You can use promise callbacks to check readyness before getting and updating
  * settings.ready().then(() => settings.updateSetting('settingName', settingValue))
  */
-require.scopes.settings =(() => {
-    var settings = {};
-    let isReady = false
-    let _ready = init().then(() => {
-        isReady = true
-        console.log("Settings are loaded")
+var settings = {};
+let isReady = false
+let _ready = init().then(() => {
+    isReady = true
+    console.log("Settings are loaded")
+})
+
+const defaultSettings = require('../data/defaultSettings')
+
+// external settings defines a function that needs to run when a setting is updated
+let isExtensionEnabled
+var externalSettings = {
+    'httpsEverywhereEnabled': function(value){ isExtensionEnabled = value }
+};
```

Comment:
If you look at the diff with `?w=1` in the URL you'll see that I added the line `let isExtensionEnabled`.

This is another one of those undefined global variables. `isExtensionEnabled` is not used anywhere else in the codebase, so I think this whole block may be good to remove. I've added a separate task for myself so I can come back to this later.